### PR TITLE
update lidar plan post

### PR DESCRIPTION
**Pull request by rkelson**
Pull request description
------------------------

### CODE REVIEW

File: _posts/2015-08-19-statewide-lidar-plan-for-utah.md [hunk 1]
```diff
@@ -33,8 +33,8 @@ tags:
 ---
 
 <p>
-    <a href="{{ "/downloads/LidarPlan.png" | prepend: site.baseurl }}" target="_blank">
```
Comment (steveoh) [inline _posts/2015-08-19-statewide-lidar-plan-for-utah.md]: can /downloads/lidarplan.png be deleted?

File: _posts/2015-08-19-statewide-lidar-plan-for-utah.md [hunk 2]
```diff
@@ -33,8 +33,8 @@ tags:
...
-        <img src="{{ "/images/LidarPlan-231x300.png" | prepend: site.baseurl }}" alt="" title="LidarPlan" width="231" height="300" class="inline-text-left" />
```
Comment (steveoh) [inline _posts/2015-08-19-statewide-lidar-plan-for-utah.md]: can imges/lidarplan-231*300.png be deleted?

File: data/index.html [hunk 3]
```diff
@@ -180,6 +176,10 @@
             <a href="{{ "/data/sgid-transportation/" | prepend: site.baseurl }}"><h5 class="text-center">Transportation</h5>
             <img class="b-lazy data-thumb" src=data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw== data-src="{{ "/images/front-page/90x90_utah-gis-highway-lrs-udot.png" | prepend: site.baseurl }}"></a>
         </div>
+              <div class="grid__col grid__col--1-of-4 grid__col--m-1-of-3 text-center">
```
Comment (steveoh) [inline data/index.html]: thanks for doing this!

Comment (rkelson) [inline data/index.html]: might as well

File: _posts/2015-08-19-statewide-lidar-plan-for-utah.md (hunk 2 above)
Comment (rkelson) [inline _posts/2015-08-19-statewide-lidar-plan-for-utah.md]: might as well

File: _posts/2015-08-19-statewide-lidar-plan-for-utah.md (hunk 1 above)
Comment (rkelson) [inline _posts/2015-08-19-statewide-lidar-plan-for-utah.md]: might as well

**Pull request merged**